beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -215,11 +216,74 @@ func (vs *Server) GetBeaconBlock(ctx context.Context, req *ethpb.BlockRequest) (
 func (vs *Server) ProposeBeaconBlock(ctx context.Context, req *ethpb.GenericSignedBeaconBlock) (*ethpb.ProposeResponse, error) {
 	ctx, span := trace.StartSpan(ctx, "ProposerServer.ProposeBeaconBlock")
 	defer span.End()
+
 	blk, err := blocks.NewSignedBeaconBlock(req.Block)
```

Comment:
doesn't blob check need to be before this ? block.NewSignedBeaconBlock doesn't handle the blockandblobs from the generic signed beacon block will fail when called.